### PR TITLE
[ISSUE #511] Polish the application.properties to solve the misunderstanding between push consumer and pull consumer

### PR DESCRIPTION
**Pull request by RongtongJin**
## What is the purpose of the change

Polish the application.properties to solve the misunderstanding between push consumer and pull consumer

## Brief changelog

Polish the application.properties to solve the misunderstanding between push consumer and pull consumer

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily. Notice, `it would be helpful if you could finish the following 5 checklist(the last one is not necessary)before request the community to review your PR`.

- [ ] Make sure there is a [Github issue](https://github.com/apache/rocketmq/issues) filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue. 
- [ ] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
- [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Write necessary unit-test(over 80% coverage) to verify your logic correction, more mock a little better when cross module dependency exist. 
- [ ] Run `mvn -B clean apache-rat:check findbugs:findbugs checkstyle:checkstyle` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.
- [ ] If this contribution is large, please file an [Apache Individual Contributor License Agreement](http://www.apache.org/licenses/#clas).

### CODE REVIEW

File: rocketmq-spring-boot-samples/rocketmq-produce-acl-demo/src/main/resources/application.properties [hunk 1]
```diff
@@ -13,10 +13,10 @@
...
 
-rocketmq.name-server=Endpoint_of_Aliware_MQ
+rocketmq.name-server=Endpoint_of_CLOUD_MQ
 rocketmq.producer.group=my-group1
 rocketmq.producer.access-key=AK
 rocketmq.producer.secret-key=SK
 
-demo.rocketmq.topic=normal_topic_define_in_Aliware_MQ
-demo.rocketmq.transTopic=transaction_topic_define_in_Aliware_MQ
+demo.rocketmq.topic=normal_topic_define_in_Cloud_MQ
```
Comment (francisoliverlee) [inline rocketmq-spring-boot-samples/rocketmq-produce-acl-demo/src/main/resources/application.properties:21]: topic， group的名字格式建议都用小写

File: rocketmq-spring-boot-samples/rocketmq-consume-acl-demo/src/main/java/org/apache/rocketmq/samples/springboot/ACLStringConsumer.java [hunk 2]
```diff
@@ -26,8 +26,8 @@
  */
 @Service
 @RocketMQMessageListener(
-    topic = "normal_topic_define_in_Aliware_MQ",
-    consumerGroup = "group_define_in_Aliware_MQ"
+    topic = "normal_topic_define_in_Cloud_MQ",
+    consumerGroup = "group_define_in_Cloud_MQ"
```
Comment (francisoliverlee) [inline rocketmq-spring-boot-samples/rocketmq-consume-acl-demo/src/main/java/org/apache/rocketmq/samples/springboot/ACLStringConsumer.java:30]: topic， group的名字格式建议都用小写

File: rocketmq-spring-boot-samples/rocketmq-consume-acl-demo/src/main/java/org/apache/rocketmq/samples/springboot/ACLStringTransactionalConsumer.java [hunk 3]
```diff
@@ -27,7 +27,7 @@
 @Service
 @RocketMQMessageListener(
     topic = "${demo.rocketmq.transTopic}",
-    consumerGroup = "group_define_in_Aliware_MQ",
+    consumerGroup = "group_define_in_Cloud_MQ",
```
Comment (francisoliverlee) [inline rocketmq-spring-boot-samples/rocketmq-consume-acl-demo/src/main/java/org/apache/rocketmq/samples/springboot/ACLStringTransactionalConsumer.java:30]: topic， group的名字格式建议都用小写

File: rocketmq-spring-boot-samples/rocketmq-consume-acl-demo/src/main/resources/application.properties [hunk 4]
```diff
@@ -16,13 +16,15 @@
 
 spring.application.name=rocketmq-consume-acl-demo
 
-rocketmq.name-server=Endpoint_of_Aliware_MQ
-rocketmq.consumer.group=my-group1
-rocketmq.consumer.topic=test
-rocketmq.topic=normal_topic_define_in_Aliware_MQ
+rocketmq.name-server=Endpoint_of_Cloud_MQ
```
Comment (francisoliverlee) [inline rocketmq-spring-boot-samples/rocketmq-consume-acl-demo/src/main/resources/application.properties:19]: 这种命名格式建议统一成小写， 下划线分隔

File: rocketmq-spring-boot-samples/rocketmq-consume-acl-demo/src/main/resources/application.properties [hunk 5]
```diff
@@ -16,13 +16,15 @@
...
+# When set rocketmq.pull-consumer.group and rocketmq.pull-consumer.topic, rocketmqTemplate will start lite pull consumer
+# If you do not want to use lite pull consumer, please do not set rocketmq.pull-consumer.group and rocketmq.pull-consumer.topic
+rocketmq.pull-consumer.group=my-group1 
+rocketmq.pull-consumer.topic=test
+rocketmq.topic=normal_topic_define_in_Cloud_MQ
 
 # properties used in application code
-demo.rocketmq.transTopic=transaction_topic_define_in_Aliware_MQ
+demo.rocketmq.transTopic=transaction_topic_define_in_Cloud_MQ
```
Comment (francisoliverlee) [inline rocketmq-spring-boot-samples/rocketmq-consume-acl-demo/src/main/resources/application.properties:27]: 这种命名格式建议统一成小写， 下划线分隔

**Review by ShannonDing — APPROVED** [review]

**Pull request merged**